core/pubnub_api_types.h

```diff
@@ -97,15 +99,21 @@ enum pubnub_res {
     /** Server reports an error in the response */
     PNR_ERROR_ON_SERVER,
     /** Proxy authentication failed */
-    PNR_AUTHENTICATION_FAILED
+    PNR_AUTHENTICATION_FAILED,
+    /** Entity api invalid parameter */
```

Comment:
1. Should this be in this PR?
2. Should we rename to `OBJECTS` ?

Reply:
It was some work to remove it, so I opted out to leave and just update as we finish up.

Reply:
Works for me.